service-mesh/app-src/apps/demo/api.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
+
+# API backend for the website.
+
+from flask import Flask, abort, jsonify, request
+import os
+import requests
+from signalfx_tracing import auto_instrument
+from signalfx_tracing.libraries import requests_config
+from . import utils
+
+
+tracer = utils.get_tracer(os.getenv('SERVICE_NAME', 'api'))
```

Comment:
Seems to me enough of this information can be provided via the standard env vars to make get_tracer unnecessary (`SIGNALFX_SERVICE_NAME`, `SIGNALFX_ENDPOINT_URL`).  If it's not possible to set the desired endpoint url var in the kubernetes yaml maybe just
`tracer = create_tracer(config=(jaeger_endpoint=utils.get_ingest() +  '/v1/trace'))`?

Ideally we'd not be directing users to assemble jaeger configs and only use create_tracer.

Reply:
Cleaned up the jaeger config